### ISSUE TITLE
run_pylint signature

### ISSUE THREAD

**Issue by dansebcar** [pylint-dev/pylint#8766]
### Question

The `run_pylint` function [is documented](https://pylint.readthedocs.io/en/v2.17.4/development_guide/api/index.html) with a variadic signature:

```python
run_pylint("--disable=C", "myfile.py")
```

But the function actually [takes one argument](https://github.com/pylint-dev/pylint/blob/v2.17.4/pylint/__init__.py#L28).

### Documentation for future user

It should say:

```python
run_pylint(["--disable=C", "myfile.py"])
```
